### ISSUE TITLE
install geonode-docker 4.1 on nginx server

### ISSUE THREAD

**Issue by eiacopini** [GeoNode/geonode#11209]
## Expected Behavior
Hi I have a server on which nginx is installed. 
I have enabled two third level domains both with certificate on port 443. 
The first third level domain points to a docker container with Directus on port 80:8055 (only this port in file config) and everything works fine even in https.
The other third level domain points to an installation of geonode 4.1 via docker.

## Actual Behavior
In the .env file I put the following configurations:
SITEURL= https://geonode.dominio.com
GEONODE_LB_HOST_IP= geonode.dominio.com
GEONODE_LB_PORT=80
PUBLIC_PORT=80
HTTP_HOST= http://geonode.dominio.com
HTTPS_HOST = https://geonode.dominio.com
HTTP_PORT=80
HTTPS_PORT=443
I am currently getting an error when doing docker-compose up -d on geonode4nginx
LETSENCRYPT_MODE=disabled

I am currently getting an error when doing docker-compose up -d on geonode4nginx
WARNING host is already in use in another container
Error 
ERROR: for geonode  Cannot start service geonode: driver failed programming external connectivity on endpoint nginx4geonode  Error starting userland proxy: Bind for 0.0.0.0:443: bind address already in use
ERROR: Encountered errors while bringing up the project

////  /etc/nginx/sites-available/geonode.dominio.com
server {
       server_name geonode.dominio.com;
       location / {
        proxy_pass http://127.0.0.1:80;
       }
    listen 443 ssl; 
    ssl_certificate /etc/letsencrypt/live/geonode.dominio.com/fullchain.pem; 
    ssl_certificate_key /etc/letsencrypt/live/geonode.dominio.com/privkey.pem; 
    include /etc/letsencrypt/options-ssl-nginx.conf;
ssl_dhparam /etc/letsencrypt/ssl-dhparams.pem
}
server {
if ($host= geonode.dominio.com) {
        return 301 https://$host$request_uri;
    } 
 server_name geonode.dominio.com;
 listen 80;
return 404;
}

////  /etc/nginx/sites-available/directus.dominio.com
server {
       server_name geonode.dominio.com;
       location / {
        proxy_pass http://127.0.0.1:8055;
       }
    listen 443 ssl; 
    ssl_certificate /etc/letsencrypt/live/directus.dominio.com/fullchain.pem; 
    ssl_certificate_key /etc/letsencrypt/live/directus.dominio.com/privkey.pem; 
    include /etc/letsencrypt/options-ssl-nginx.conf;
ssl_dhparam /etc/letsencrypt/ssl-dhparams.pem
}
server {

if ($host= directus.dominio.com) {
        return 301 https://$host$request_uri;
    } 
 server_name directus.dominio.com;
 listen 80;
return 404;
}

In site avaiable/enable i have also default config.
Thank you for all suggest!!!

## Specifications

  - GeoNode version: 4.1 latest
  - Installation type (vanilla, geonode-project): geonode-project
  - Installation method (manual, docker): docker
  - Platform: ubuntu
  - Additional details: existing nginx on the server

**Comment by giohappy** [comment]
Looking at the error I suspect that you're trying to start the Nginx provided by GeoNode's own docker-compose configuration, while another Nginx is already running on your machine. Is it the case?

**Comment by eiacopini** [comment]
@giohappy Yes if i stop the nginx server i can install geonode but after i have the problem with the port.

**Comment by giohappy** [comment]
>Yes if i stop the nginx server i can install geonode but after i have the problem with the port.

when you stop which Nginx? The one one the machine?

**Comment by eiacopini** [comment]
Yes the nginx on the machine.
